manifests/noderesourcetopology/crd.yaml

```diff
@@ -1,9 +1,10 @@
+
+---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    api-approved.kubernetes.io: https://github.com/kubernetes/enhancements/pull/1870
-    controller-gen.kubebuilder.io/version: v0.6.0
```

Comment:
restore this line, and replace v0.6.0 with v0.6.2

Reply:
I've restored the line and put v0.6.2 as new value

Reply:
The last falling piece, I again misread the change. Fixed it and I can see PR checks have also passed now. Thanks @Huang-Wei for your valuable reviews 🙂